src/types.ts

```diff
@@ -1,6 +1,7 @@
-import {codemods} from 'module-replacements-codemods';
 import type {FileSystem} from './file-system.js';
 
+type Codemods = typeof import('module-replacements-codemods').codemods;
```

Comment:
can we not just use a regular import for this?

if you `import type {codemods}`, it'll be the type representing `codemods` i'd hope 🤔  then you could `typeof that`?

not sure if thats how it works though